src/aarch64/init.h

```diff
@@ -30,6 +30,9 @@ common_init (struct cursor *c, unsigned use_prev_instr)
 {
   int ret, i;
 
+  for (i = UNW_AARCH64_PSTATE + 1; i < UNW_AARCH64_V0; ++i)
```

Comment:
UNW_AARCH64_VG falls in that range so the ones that are set are sparse. It would be a pain if further ones are added to keep that loop up to date so I just moved it to the beginning so everything is zeroed before being set.